modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -4903,7 +4903,7 @@ Point LineIterator::pos() const
 {
     Point p;
     p.y = (int)((ptr - ptr0)/step);
-    p.x = (int)(((ptr - ptr0) - p.y*step)/elemSize);
+    p.x = (int)(((ptr - ptr0) - (long)p.y*step)/elemSize);
```

Comment:
`long` is still [32-bit on Win32](https://en.cppreference.com/w/cpp/language/types).

Consider using `int64` here (or `long long int`).